eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/FileSizeExtension.php

```diff
@@ -100,7 +100,9 @@ public function sizeFilter($number, $precision)
         }
         $formatter = new NumberFormatter($this->getLocale(), NumberFormatter::PATTERN_DECIMAL);
         $formatter->setPattern($formatter->getPattern() . ' ' . $this->translator->trans($this->suffixes[$i]));
-        $return = $formatter->format(round($number, $precision));
+        $formatter->setAttribute(NumberFormatter::MAX_FRACTION_DIGITS, $precision);
+
+        $return = $formatter->format($number);
 
         return $return;
```

Comment:
Just noticed it should be:
```suggestion
        return $formatter->format($number);
```

Reply:
Fixed during rebase.